sequencer/api/migrations/V38__upgrade_certificate.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE upgrade_certificate (
+    id bool PRIMARY KEY DEFAULT true,
+    data BYTEA
+);
+REVOKE DELETE, TRUNCATE ON upgrade_certificate FROM public;
```

Comment:
So this wouldn't allow garbage_collect() to work as we are revoking the delete permissions

Reply:
yes we don't need to delete this certificate, I deleted the code for that after talking with @ss-es